### ISSUE TITLE
Forced colour mode bugs

### ISSUE THREAD

**Issue by querkmachine** [querkmachine/beeps.website#85]
Various issues in forced colour modes, such as Windows High Contrast Mode:

## Links with underlines are unreadable

Forced colour modes make the underline appear in the same colour as the link text. The chunky, overlapping underline style used means that the underline now significantly obscures the text, making it difficult or impossible to read.

<img width="560" alt="Screenshot of part of the website homepage with lots of text and links. The links are yellow and appear to have a yellow box covering them up partially." src="https://github.com/user-attachments/assets/8c5d6b1f-2a96-4eca-b476-fe2153af67af">

- [x] Fixed

## Table of contents arrow isn't visible

The arrow on the table of contents component isn't visible in forced colours mode.

<img width="352" alt="Screenshot of the table of contents, which appears entirely as yellow text with no obvious means of expanding or collapsing it." src="https://github.com/user-attachments/assets/7c3afb4a-3705-47c5-8ac7-913bd86e11f1">

- [x] Fixed

## Colour swatches have their colours overridden

Places where colour is necessary for context, such as on the brand page and Ash's reference sheet, does not persist those colours in forced colour modes.

<img width="722" alt="Screenshot of the brand page's swatches, which are all rendering as black." src="https://github.com/user-attachments/assets/894df2ba-5f3e-4b8f-88c1-2c6df0f454dc">

<img width="1084" alt="Screenshot of a horizontal list of colour swatches. All of the swatches appear in white." src="https://github.com/user-attachments/assets/bae72281-6a89-4c69-b4be-af24b41f35d1">

- [x] Fixed

## Webring icons don't respect forced text colours

The SVG icons used to represent webrings don't adapt to forced colour modes and continue to render according to the dark/light mode setting.

<img width="635" alt="Screenshot of webrings section, with icons appearing in dark grey and purple on a black background." src="https://github.com/user-attachments/assets/8c4c4c66-7816-4de1-9054-25a5eb5a8063">

- [x] Fixed

## Spots don't respect forced colours

'Spots' areas continue to render according to the dark/light mode setting, without regard for the forced colour mode in use.

- [x] Fixed

## Active tab isn't visually discernible 

The active tab within a set of tabs appears identically to inactive tabs.

<img width="422" alt="Screenshot of a list of tabs, all of them appearing as yellow text inside of a yellow box." src="https://github.com/user-attachments/assets/1cb22a28-3bc7-4d0a-acec-1bb5713d8012">

- [x] Fixed

## Image differ drag handle doesn't have visible marks

The drag handle typically has a few parallel lines as a subtle indicator that it can be dragged left and right. These marks aren't visible in forced colour modes.

- [x] Fixed
